Task08.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+
+void printMatrix(int a[25][25], int row, int column) {
+    for(int i = 0; i < row; i++) {
+        for(int j = 0; j < column; j++) {
+            std::cout << a[i][j] << " ";
+        }
+        std::cout << std::endl;
+    }
+}
+
+void initializeMatrix(int a[25][25], int row, int column) {
+    std::cout << "Please fill in your matrix" << std::endl;
+    for(int i = 0; i < row; i++) {
+        for(int j = 0; j < column; j++){
+        std::cin >> a[i][j];
+    }
+    }
+}
+
+void multiplyAndPrint(int a[25][25], int b[25][25], int rowA, int colA_rowB, int colB) {
```

Comment:
Note: Single responsibility principle - One function should do one thing => one function should only multiply funcs and one function should only print arrays